src/LanguageFactory.hack

```diff
@@ -58,9 +58,9 @@ class HTMLPurifier_LanguageFactory {
 	*/
 	public static function instance(?HTMLPurifier_LanguageFactory $prototype = null): HTMLPurifier_LanguageFactory {
 		$instance = null;
-		if ($prototype !== null) {
+		if ($prototype is nonnull) {
 			$instance = $prototype;
-		} else if ($instance === null || $prototype == true) {
+		} else if ($instance is null || $prototype == true) {
```

Comment:
hmm suppose this `||  prototype == true` isn't needed since getting past the first `if` indicates that `$prototype` is null 😅